phac-nml/irida-next
# Group Attachments: Destroy

## What does this PR do and why?
This PR adds file deletion to group attachments.

## Screenshots or screen recordings
[Screencast from 2024-09-20 10:02:46 AM.webm](https://github.com/user-attachments/assets/040bfc53-179b-4c95-98a0-0e614851b7c0)

## How to set up and validate locally
1. Navigate to a group `Files` page.
2. Upload files if needed (including paired end files)
3. Delete the files using the `Delete` link within each row (paired end files should both delete with a single action)
4. Ensure the table updates and no longer includes the deleted file(s)

Other test:
5. Verify deleting files from projects still works. It shares logic with group file deletion.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
